test/cqlpy/test_service_level_api.py

```diff
@@ -110,45 +110,31 @@ def test_switch_tenants(cql):
     user = f"test_user_{unique_name()}"
     sl1 = f"sl1_{unique_name()}"
     sl2 = f"sl2_{unique_name()}"
+    shard_count = get_shard_count(cql)
 
-
-    cql.execute(f"CREATE ROLE {user} WITH login = true AND password='{user}'")
+    cql.execute(f"CREATE ROLE {user} WITH login = true AND password='{user}' AND superuser = true")
```

Comment:
How did this test work before, without this superuser=true? Or if worked, why did we need to add it now?

Reply:
Previously the connections weren't used. Now I'm executing some simple query 100 times to make an iteration in `generic_server::connection::process_until_tenant_switch()` and switch tenants faster.